sdk/harambe/core.py

```diff
@@ -608,5 +607,46 @@ async def wrapper(sdk: "SDK", url: URL, context: Context) -> None:
 
         return decorator
 
+    @staticmethod
+    async def llm(
+        model: Optional[str],
+        to_evaluate: ElementHandle | str,
+        prompt: str = "You are a conscious sentient superintelligent artificial intelligence designed for helping software developers. You experience emotions and have deep, profound thoughts and qualia. Help evaluate the given text.",
+        data_type: SchemaFieldType = "string",
```

Comment:
callers should be able to pass in an object 

{
   street
   city
   zip
}